atramhasis/scripts/import_file.py

```diff
@@ -262,10 +238,9 @@ def main(argv=sys.argv):
     if args.cs_uri:
         cs_uri = args.cs_uri
         cs_label = args.cs_label if args.cs_label else input_name.capitalize()
-        cs = create_conceptscheme(cs_label, cs_uri)
-    else:
-        cs = create_conceptscheme_from_skos(provider.concept_scheme)
-    provider_to_db(provider, cs, session)
+        args.concept_scheme = create_conceptscheme(cs_uri, cs_label)
+    provider = file_to_provider_function(**vars(args))
```

Comment:
There's also a `file_to_provider_function` call on line 237. I assume that one should be deleted.